.changeset/khaki-wolves-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vygruppen/spor-input-react": patch
+---
+
+Change front page illustation and fix choichips icon to be checked state.
```

Comment:
Når vi innfører breaking changes burde vi skrive inn en migreringsguide for brukerne.

```suggestion
"@vygruppen/spor-input-react": minor
---

Breaking change: The `ChoiceChip`'s `icon` prop now requires an object that requires both a default icon and a checked icon.

Previously, the code would be:

<ChoiceChip icon={<SomeIcon />} />

Now, the code would be:

<ChoiceChip
  icon={{
    default: <SomeOutlineIcon />,
    checked: <SomeFillIcon />
  }}
/>
```